bukuserver/server.py

```diff
@@ -98,6 +99,8 @@ def create_app(db_file=None):
     bukudb = BukuDb(dbfile=db_file)
     app.config['FLASK_ADMIN_SWATCH'] = (os.getenv('BUKUSERVER_THEME') or 'default').lower()
     app.config['BUKUSERVER_LOCALE'] = os.getenv('BUKUSERVER_LOCALE') or 'en'
+    _dir = os.path.dirname(os.path.realpath(__file__))
+    app.config['SWAGGER'] = {'title': 'Bukuserver API', 'doc_dir': os.path.join(_dir, 'apidocs')}
```

Comment:
This library actually supports SwaggerUI v3 (=OpenAPI), but its UI seemed more clunky so I've decided to keep using v2